config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get 'message/index', to: 'message#index'
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [Optional] Indicating the root routes is important-- This prevents the default Rails set up page from displaying :+1: 
![Screenshot from 2023-10-08 05-25-39](https://github.com/yosaddis/hello-rails-back-end/assets/63087491/fc0a3e21-87e1-4841-997f-6f6f998c37a9)